wolvic/browser/vr/wvr_manager.cc

```diff
@@ -117,8 +117,6 @@ device::Gamepad ToGamepad(const mozilla::gfx::VRControllerState& controller) {
     }
   }
 
-  // TODO: fill in hand tracking data here
-
```

Comment:
I think we can leave the comment here, but changing it a bit. What we eventually (not for this patch) need to do here is to fill in the Gamepad data when the device has no physical controllers. We might include a link to https://www.w3.org/TR/webxr-gamepads-module-1/#gamepad-api-integration